tests/helpers/perf_run_helper.rb

```diff
@@ -262,7 +262,10 @@ def scale_setup(gatling_scenario, simulation_id, gatlingassertions)
         puts
 
         # restart using master_manipulator
-        restart_puppet_server(master)
+        masters = select_hosts(roles: %w[master compile_master])
```

Comment:
It seems confusing to call the set of hosts 'masters' and then each host 'compiler'. It might be less confusing to just call the set 'hosts_to_restart' and each host 'host'.

Reply:
These are being restarted specifically because they nodes running puppetserver.  Using generic labels would not convey the intended classification for restarting them.
I agree that the current mixed use of master(s)/compiler is confusing.  Since this block of code is intended to restart puppet servers how about the following?
```
puppet_servers = select_hosts(roles: %w[master compile_master])
puppet_servers.each do |puppet_server|
  restart_puppet_server(puppet_server)
end
```

Reply:
puppet_servers works for me.  But I don't love having variables puppet_server and puppet_servers.  With just an s difference.  I would call the loop var either server or host.